pkg/k8shandler/logstore.go

```diff
@@ -255,7 +254,13 @@ func (cr *ClusterLoggingRequest) newElasticsearchCR(elasticsearchName string, ex
 
 	indexManagementSpec := indexmanagement.NewSpec(logStoreSpec.RetentionPolicy)
 
-	es := cr.emptyElasticsearchCR(elasticsearchName)
+	var es *elasticsearch.Elasticsearch
+	if existing != nil && existing.Name != "" {
+		// using existing es if a valid one exists
+		es = existing.DeepCopy()
+	} else {
+		es = cr.emptyElasticsearchCR(elasticsearchName)
+	}
```

Comment:
i'm not following why this is the fix. this makes it seem like something in our create/update logic is broken and this is a bandaid. 

Maybe I'm not understanding the root issue though?

Reply:
I'm not following either.  Typically the pattern is:
* new proto
* Create(proto)
* if fail and already exists:
* get instance
* update instance to equal proto (or the parts as needed